src/H5Iint.c

```diff
@@ -887,26 +890,32 @@ H5I_remove(hid_t id)
 } /* end H5I_remove() */
 
 /*-------------------------------------------------------------------------
- * Function:    H5I_dec_ref
+ * Function:    H5I__dec_ref
  *
  * Purpose:     Decrements the number of references outstanding for an ID.
  *              This will fail if the type is not a reference counted type.
  *              The ID type's 'free' function will be called for the ID
  *              if the reference count for the ID reaches 0 and a free
  *              function has been defined at type creation time.
  *
+ * Note:        Allows for asynchronous 'close' operation on object, with
+ *              token != H5_REQUEST_NULL.
+ *
  * Return:      Success:    New reference count
  *              Failure:    -1
  *
+ * Programmer:  Houjun Tang
+ *              Oct 21, 2019
+ *
  *-------------------------------------------------------------------------
```

Comment:
Also nothing against Tang but you should really stop adding more of these programmer comments, it really does not make any sense. I could understand if that were a whole new module or something but for a single function like that it just adds more confusion that anything else.

Reply:
It would be better in my opinion that you add him as a co-author of your PR rather than adding names in the middle of the code like that. You can do that by adding: `Co-authored-by: name <name@example.com>` in your commit message.

Reply:
Sure, fixed.   (Will include Tang as co-author)